parsec/core/gui/instance_widget.py

```diff
@@ -44,6 +43,19 @@ async def _do_run_core(config, device, qt_on_ready):
                 qt_on_ready.emit(core, core_jobs_ctx)
 
 
+def macfuse_installation_widget(window):
```

Comment:
A more explicit name would be welcome, like `ensure_macfuse_available_or_show_dialogue`